src/natcap/invest/forest_carbon_edge_effect.py

```diff
@@ -618,17 +617,17 @@ def mask_non_forest_op(lulc_array):
                 each forest LULC code is in `forest_codes`.
         Returns:
             numpy.ndarray with the same shape as lulc_array. All pixels are
-                0 (forest), 1 (non-forest), or 255 (nodata).
+                0 (forest), 1 (non-forest), or -1 (nodata).
         """
         non_forest_mask = ~numpy.isin(lulc_array, forest_codes)
         nodata_mask = lulc_array == lulc_nodata
-        # where LULC has nodata, set value to forest_mask_nodata value (255)
+        # where LULC has nodata, set value to nodata value (-1)
         # where LULC has data, set to 0 if LULC is a forest type, 1 if it's not
-        return numpy.where(nodata_mask, forest_mask_nodata, non_forest_mask)
+        return numpy.where(nodata_mask, NODATA_VALUE, non_forest_mask)
 
     pygeoprocessing.raster_calculator(
         [(base_lulc_raster_path, 1)], mask_non_forest_op,
-        target_non_forest_mask_path, gdal.GDT_Byte, forest_mask_nodata)
+        target_non_forest_mask_path, gdal.GDT_Byte, NODATA_VALUE)
```

Comment:
Hmm. Will this be a problem that we're returning a `gdal.GDT_Byte` but using a `-1` nodata value? Since we're not setting metadata for `IMAGE_STRUCTURE` of the raster, I'm thinking GDAL will assume Unsigned Byte. Thoughts?

Reply:
That's a good point! I just wanted to use a consistent nodata value throughout the model. I'll use type int8 instead

Reply:
Just going to leave it as is with the 255 value and byte type